.envrc

```diff
@@ -0,0 +1,2 @@
+source venv/bin/activate
```

Comment:
leftover file?

Reply:
oooops, good catch. I usually add it to `.git/info/exclude` but forgot in this repo

Removed it as well as `noxfile.py` in 4be77731a228